app/models/derived_variables/lettings_log_variables.rb

```diff
@@ -1,4 +1,5 @@
 module DerivedVariables::LettingsLogVariables
+  # this is opaque enough to deserve a comment
```

Comment:
@kosiakkatrina it sounded when we discussed that you had an idea of why this is this way. I feel this could do with a comment so at least hopefully the next person to engage with this code knows that we all have sympathy with them and the sinking feeling in their stomach

Reply:
So I think the column that CDS cares about is `renttype`, which has these options:
```
    "1" => "Social Rent" 
    "2" => "Affordable Rent" 
    "3" => "Intermediate Rent"
```
 But we ask `rent_type` question which is something we defined and it has these options:
```
    "1" => "Affordable Rent" 
    "2" => "London Affordable Rent" 
    "4" => "London Living Rent" 
    "3" => "Rent to Buy" 
    "0" => "Social Rent" 
    "5" => "Other intermediate rent product" 
```

We could potentially add something like this here:
```
RENT_TYPE_MAPPING = { 
0 => 1, # "Social Rent"  =>  "Social Rent" 
1 => 2, # "Affordable Rent" => "Affordable Rent" 
2 => 2, # "London Affordable Rent"  =>  "Affordable Rent" 
3 => 3, # "Rent to Buy"  => "Intermediate Rent"
4 => 3, #  "London Living Rent"  => "Intermediate Rent"
5 => 3 # "Other intermediate rent product"  => "Intermediate Rent"
 }.freeze
```